iouring_test.go

```diff
@@ -0,0 +1,45 @@
+package iouring
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+func testMany(nreqs uint, t *testing.T) {
+	f, err := os.Open("/dev/zero") // For read access.
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer f.Close()
+	iour, err := New(nreqs)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer iour.Close()
+
+	fd := int(f.Fd())
+	for iter := 0; iter < 10; iter++ {
```

Comment:
Why do we need to loop 10 times? Will less compromise our purpose? 🤔

Reply:
The idea is to run SubmitRequests more than once. We write to /dev/zero, this has to be fast, so I think it doesn't matter in this case whether it is 10 or 2.